lab_1/main.py

```diff
@@ -49,7 +81,13 @@ def get_top_n_words(freq_dict: dict, top_n: int) -> list:
     top_n = 1
     --> ['happy']
     """
-    pass
+    top = []
+    if isinstance(freq_dict, dict):
```

Comment:
Usually, there are checks for input arguments and a function body - separated.
Try to split them and get rid of nesting.

Reply:
I don't understand what you mean. We need that check to work with freq_dict, otherwise we get a mistake

Reply:
You do need that check, yes.
Here what I mean:

```suggestion
    if not isinstance(freq_dict, dict):
        return []
    main body of the function
```

As you can see - they are on the same level - and function body is not nested.
It is much easier to read.